src/coreclr/vm/arm/gmscpu.h

```diff
@@ -48,9 +40,6 @@ struct MachState {
 	//
 	// 2) HelperMethodFrame::UpdateRegdisplay is invoked by our StackWalker that initializes
```

Comment:
These comments makes me think `MachState` is also up for sweep?

Reply:
Yep